app/_hub/kong-inc/jwt-signer/index.md

```diff
@@ -34,747 +38,706 @@ kong_version_compatibility:
+      description: |
+        Tokens signed with HMAC algorithms such as `HS256`, `HS384`, or `HS512` are not
+        accepted by default. If you need to accept such tokens for verification,
+        enable this setting.
+    - name: enable_instrumentation
+      required: false
+      default: false
+      datatype: boolean
+      description: |
+        When you are experiencing problems in production and don't want to change
+        the logging level on Kong nodes, which requires a reload, use this
+        parameter to enable instrumentation for the request. The parameter writes
+        log entries with some added information using `ngx.CRIT` (CRITICAL) level.
+    - name: access_token_issuer
+      required: false
+      default: kong
+      datatype: string
+      description: |
+        The `iss` claim of a signed or re-signed access token is set to this value.
+        Original `iss` claim of the incoming token (possibly introspected) is
+        stored in `original_iss` claim of the newly signed access token.
+    - name: access_token_keyset
+      required: false
+      default: kong
+      datatype: string
+      description: |
+        Selects the private key for access token signing.
+    - name: access_token_jwks_uri
+      required: false
+      default:
+      datatype: string
+      description: |
+        If you want to use `config.verify_access_token_signature`, you must specify
+        the URI where the plugin can fetch the public keys (JWKS) to verify the
+        signature of the access token. If you don't specify a URI and you pass a
+        JWT token to the plugin, then the plugin responds with
+        `401 Unauthorized`.
+    - name: access_token_request_header
+      required: false
+      default: authorization
+      datatype: string
+      description: |
+        This parameter tells the name of the header where to look for the access token.
+        By default, the plugin searches it from `Authorization: Bearer <token>` header
+        (the value being magic key `authorization:bearer`). If you don't want to
+        do anything with `access token`, then you can set this to `null` or `""` (empty string).
+        Any header can be used to pass the access token to the plugin. Two predefined
+        values are `authorization:bearer` and `authorization:basic`.
+    - name: access_token_leeway
+      required: false
+      default: 0
+      datatype: number
+      description: |
+        Adjusts clock skew between the token issuer and Kong. The value
+        is added to the token's `exp` claim before checking token expiry against
+        Kong servers' current time in seconds. You can disable access token
+        `expiry` verification altogether with `config.verify_access_token_expiry`.
+    - name: access_token_scopes_required
+      required: false
+      default:
+      datatype: array of string elements
+      description: |
+        Specify the required values (or scopes) that are looked from a
```

Comment:
@bungle I've edited most of this but am not sure about phrasing for:

> that are looked from a

This is a global review comment for wherever this occurs in the doc.